molgenis/molgenis-emx2
# feat(directory): Orphanet hierarchy

how to test:
https://preview-emx2-pr-3480.dev.molgenis.org
- Model + DiseaseTypes update: Go to the preview, login and check table [DiseaseTypes](https://preview-emx2-pr-3480.dev.molgenis.org/directory-demo/tables/#/DiseaseTypes). Please note that the Diagnosis available filter is broken (as expected) due to these changes.

todo:
- [na] updated docs in case of new feature
- [na] added tests
